src/Hl7.Fhir.Core.Tests/Serialization/SerializationTests.cs

```diff
@@ -631,18 +645,18 @@ public void SummarizeSerializingTest()
         public void DateTimeOffsetAccuracyTest()
```

Comment:
I think these changes were not supposed to be part of this PR?

Reply:
Yes, they were not. My visual studio doesn't see them at all.